tutorial/part2/chapter6.md

```diff
@@ -79,9 +79,11 @@ if __name__ == '__main__':
 
 [Click to download the complete code file for this chapter](https://raw.githubusercontent.com/open-resources/dash_curriculum/main/tutorial/part2/ch6_files/chapter6_fin_app.py)
 
-## 6.1 Where to import data within Dash apps
-The data which is imported into Dash apps will be used by multiple objects: Dash components, callbacks, tables, layout, etc.
-For this reason, we recommend importing the data before initialising the app, right above this line of code:
+## 6.1 Where to read in the data?
```

Comment:
I have changed this wording in general to make it clear that we are simply reading in data and not doing anything fancy that might be implied when using words such as "import" and "upload"